m4_sessions/1706-inning/group_two.md

```diff
@@ -1,9 +1,39 @@
 # Group Two
-
-### Session Title:
-
+ 
+### Session Title: Navigating the Attention Economy
+ 
 ### Session Outcomes: 
-
+ 
+Students should be able to do the following:
+ 
+* Describe the “Attention Economy”
+* Identify the tools used by companies and technologies to capture user attention and how that generates revenue
+* As a user, be able to use technology with and understanding of how they’re being subjected to the “attention economy”
+* As a developer, understand your role in developing technologies that contribute to, or take away from, creating tools that pull in users
+ 
 ### Session Outline:
 
-### Names of group members:
+* Introductions (5 mins)
+  * Have everyone say their name and mod. 
+* Introduce the Attention Economy (15 mins)
+  * Mark to introduce topic and what it is (3 mins)
+  * Stats!?
+  * Watch clips from 60 mins 
+  * Podcast clips 
+* Group Discussion Time (20 mins)
+  * Group students into 5 groups (max) and assign each a company - ask them to do research on the company’s mission and discuss first in your small group how effective they’ve been in meeting their mission, how are they contributing to the attention economy
+  * 5 Companies:
+    * Facebook
+    * Snapchat
+    * Meditation Company?? 
+    * Google??
+    * Amazon
```

Comment:
Could be cool to have groups write notes on chart paper and share whole group or share in a slack channel.  You will have the participant names the week of the session so it would be easy to start a channel. I think this is a great application and will be a really great discussion.